website/content/docs/concepts/security/permissions/assignable-permissions.mdx

```diff
@@ -24,33 +24,51 @@ know.
 
 #### Subactions
 
-Starting in Boundary 0.1.6, some subactions are supported. These actions have a
-format `top_level_action:subaction`, such as `read:self`. Being granted the top
-level action infers being granted all subactions. Thus, if a grant conveys
-`read`, it also matches the API actions `read` and `read:self`. However, if a
-grant conveys `read:self`, it will match the API action `read:self` but will not
-match `read`.
-
-#### The `no-op` Action
-
-Starting in Boundary 0.2.1 there is an action that can be granted called
-`no-op`. As might be apparent, `no-op` is not used for any real action in
-Boundary; the purpose of this action is for listing visibility. Boundary only
-shows resources in the output of a `list` action for which a user has at least
-one granted action. Thus, without `no-op`, in order for a resource to be visible
-in a `list` action, a different action such as `read` would have to be granted
-to a user. This could result in exposing more information than desired,
-especially in the case of listing scopes and authentication methods so that
-users can perform initial authentication to Boundary (that is, granting access
-to `u_anon`).
+Some subactions are supported. These actions have a format
+`top_level_action:subaction`, such as `read:self`. Being granted the top level
+action infers being granted all subactions. Thus, if a grant conveys `read`, it
+also matches the API actions `read` and `read:self`. However, if a grant conveys
+`read:self`, it will match the API action `read:self` but will not match `read`.
+
+#### The `no-op` Action and Listing Visibility
+
+There is an action that can be granted called `no-op`. As might be apparent,
+`no-op` is not used for any real action in Boundary; the purpose of this action
+is for listing visibility. Boundary only shows resources in the output of a
+`list` action for which a user has at least one granted action. Thus, without
+`no-op`, in order for a resource to be visible in a `list` action, a different
+action such as `read` would have to be granted to a user. This could result in
+exposing more information than desired, especially in the case of listing scopes
+and authentication methods so that users can perform initial authentication to
+Boundary (that is, granting access to `u_anon`).
 
 By granting the `no-op` action to users, they can see the resources in the
 output of a list command without needing other capability grants as well.
 
+### Anonymous User Restrictions
+
+Starting in Boundary 0.9.0, there are severe limits placed on the actions
+allowed to be assigned to the anonymous user `u_anon`. This is meant to avoid
+situations where an operator accidentally or mistakenly adds the anonymous user
+to a role that provides more privileges than might be intended for
+unauthanticated users.
```

Comment:
spelling: unauthenticated

Reply:
You know, it's weird -- I have the spell check plugin installed and sometimes it just....doesn't highlight things 😠